components/list/test/list-item-placement-marker.visual-diff.js

```diff
@@ -0,0 +1,39 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-list-item-placement-marker', () => {
+
+	const visualDiff = new VisualDiff('list-item-placement-marker', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/list/test/list-item-placement-marker.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	beforeEach(async() => await visualDiff.resetFocus(page));
```

Comment:
I think we can remove this from these tests.  In some tests where we capture screenshots of components in their focused states, we will use this helper to reset the focus between tests if the page is not being reloaded.  In this case, focus is not being applied to the component, so this isn't needed.